### ISSUE TITLE
BA97

### ISSUE THREAD

**Issue by datedefult** [allegro/allegro-api#9105]
I'm trying to use the sale/badges interface to rack badge campaigns,but return me '[{'code': 'BA97', 'messages': [{'text': 'You do not meet the condition regarding the required country of tax residence.', 'link': None}]'.Does this mean that I can't upload this badge campaign via api at my location?

**Comment by datedefult** [comment]
One of them is `6a9fa303-9fc3-4f75-bc37-1cceff9c7fa9`

**Comment by datedefult** [comment]
`aade22be-b71f-41d4-bb14-17e1038985d3`

**Comment by datedefult** [comment]
I am participating in` {'id': 'SALE_CATEGORY_ALLDEALS_API_202406_PL', 'name': 'Allegro Days API 03.06-09.06.2024'} `through the interface and it returns` {'status': 'WAITING_FOR_ PUBLICATION', 'rejectionReasons': []}`, does it mean that I successfully participated in the event?But I can see from the admin backstage that it's still off-limits.
![image](https://github.com/allegro/allegro-api/assets/103082878/a9aa8112-476a-4153-9c19-87d9f4d70656)
![image](https://github.com/allegro/allegro-api/assets/103082878/d748d6a2-d971-48fe-8754-dd9bc989fd28)
Because if I declare the activity via `{'id': 'SALE_CATEGORY_ALLDEALS_202406_PL', 'name': 'Allegro Days 03.06-09.06.2024'}`, the result returns`{'status': 'DECLINED', 'rejectionReasons': [{'code': 'BA97', 'messages': [{'text': 'You do not meet the condition regarding the required country of tax residence.', 'link': None}]}]}`

**Comment by Lukasz-Zurek** [comment]
Can you give me your login? If your base marketplace is other than PL, then you should assign to other campaign.

**Comment by datedefult** [comment]
login is `Aanizabawki`

**Comment by Lukasz-Zurek** [comment]
@datedefult I see that you have already applied to _Allegro Days API 03.06-09.06.2024_, "id": "SALE_CATEGORY_ALLDEALS_API_202406_PL" , which has been already added today early in the morning, e.g. for these offers: 15186095900, 14954758560, 15185923189. 

Previously you've been trying to apply to a different campaign "SALE_CATEGORY_ALLDEALS_202406_PL", without "API", this is why it has been rejected.

**Comment by datedefult** [comment]
So items uploaded through the api with a status of `WAITING_FOR_ PUBLICATION` will automatically be uploaded at the start time of the event to participate in the event right?

**Comment by Lukasz-Zurek** [comment]
@datedefult Yes, WAITING_FOR_PUBLICATION status means that the offers have been positively submitted to Allegro Days campaign, and this "Allegro Days" badge is waiting for publication (the offer will be marked with the special badge). It will be changed to ACTIVE status as soon as it starts.
